javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -228,6 +229,20 @@ public static boolean isFinal(Element element) {
         return element.getModifiers().contains(Modifier.FINAL);
     }
 
+    /**
+     * Returns true if the element is an anonymous constructor.
+     *
+     * @return true if the element is an anonymous constructor
+     */
+    public static boolean isAnonymousConstructor(Element element) {
```

Comment:
Yes, I did, but I didn't finnd other potential uses except the two places we mentioned in this pr. However, It is worth to have this method because it is conviniennt and maybe others will use it. I will do it in a separate pr.